index.html

```diff
@@ -98,9 +125,33 @@ <h2>What have you achieved since Jan 2017?<br/>
             };
         })();
 
+        var salaryCount = (function () {
+            var monthlyBaseSalaryFromApr2016 = 49000 / 12;
+            var monthlyBaseSalaryFromApr2017 = 49500;
+            var monthlyBaseSalaryFromApr2018 = 50000 / 12;
+            var noOfMLAs = 90;
+
+            //Cut of initial 15%
+            //var monthlyBaseSalaryAfterCutNov2018 = ((monthlyBaseSalaryFromApr2017 / 100) * 0.85)  / 12;
+            //var monthsSince = new Date(2017,0,16)
+
+            return {
+                totalPay: function() {
+                   return ((monthlyBaseSalaryFromApr2016 * 2) + (monthlyBaseSalaryFromApr2017) + (monthlyBaseSalaryFromApr2018 * 7)) * noOfMLAs;
+                   
+                },
+
+                perMLA: function(){
+                    return Math.round((this.totalPay() / noOfMLAs));
+                }
+            };
+        })();
+
         document.addEventListener("DOMContentLoaded", function () {
             document.getElementById("days-container").innerHTML = noGov.totalDays();
             document.getElementById("record-container").innerHTML = noGov.daysToRecord();
+            document.getElementById("salary-container").innerHTML = "£" + (salaryCount.totalPay() / 1000000).toFixed(2) + "m";
```

Comment:
May be good to swap the £ symbol for &pound; (two lines, I think) - I notice some encoding issues on my laptop ("Â£" appears) that this would solve?